src/main/java/uk/gov/justice/hmpps/prison/api/resource/PrisonResource.kt

```diff
@@ -39,11 +41,14 @@ class PrisonResource(private val bookingService: BookingService) {
       content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
     ),
   )
-  @Operation(summary = "Details of the offenders and sentences active at a particular establishment")
-  @GetMapping(name = "/{establishmentId}/bookings")
+  @Operation(summary = "Details of the active offender booking and calculable sentences at a particular establishment")
+  @PreAuthorize("hasRole('ROLE_RELEASE_DATE_MANUAL_COMPARER') and hasAuthority('SCOPE_read')")
```

Comment:
don't need `SCOPE_read` here as that is the default anyway

Reply:
also don't need the `ROLE_` prefix in the `PreAuthorize`

Reply:
Will do :) This is my first prison API change, so the implementation is a little unfamiliar